index.js

```diff
@@ -9,27 +9,31 @@ const clean = version => version.split('.').length === 2 ? `${version}.0` : vers
 
 const getVersion = () => {
 	if (!isMacos) {
-		throw new Error('Requires macOS');
+		return;
 	}
 
 	if (!version) {
 		const file = fs.readFileSync('/System/Library/CoreServices/SystemVersion.plist', 'utf8');
 		const matches = /<key>ProductVersion<\/key>[\s\S]*<string>([\d.]+)<\/string>/.exec(file);
 
 		if (!matches) {
-			throw new Error('Couldn\'t find the macOS version');
+			return;
 		}
 
 		version = matches[1];
 	}
 
-	return clean(version);
+	if (version) {
+		return clean(version);
+	}
 };
 
 module.exports = getVersion;
 
 const x = module.exports;
 
+x.isMacos = isMacos;
```

Comment:
Should be `.isMacOS` and should be documented and tested

Reply:
The casing was following conventions you already has in the repo:

https://github.com/sindresorhus/macos-version/blob/master/index.js#L11

Reply:
I can write a test for it but just so you know the logic was in the codebase prior to this commit:

https://github.com/sindresorhus/macos-version/blob/master/index.js#L5